Dockerfile

```diff
@@ -28,12 +28,8 @@ RUN dnf install -y maven which rpm-build panc ncm-lib-blockdevices \
   perl-Net-OpenStack-Client perl-NetAddr-IP perl-REST-Client \
   perl-Set-Scalar perl-Text-Glob cpanminus gcc wget \
   perl-Git-Repository perl-Data-Structure-Util \
-  perl-Test-Quattor aii-ks procps-ng
+  perl-Test-Quattor aii-ks procps-ng rpm-build
 
 # quattor tests should not be run as root
 RUN useradd --user-group --create-home --no-log-init --home-dir /quattor_test quattortest
-USER quattortest
```

Comment:
building RPMs (or running the test suite) are not recommend to do as root?

Reply:
Absolutely, but changing USER breaks GitHub actions

> Changing USER breaks standard GitHub actions which require root.
See https://docs.github.com/en/actions/sharing-automations/creating-actions/dockerfile-support-for-github-actions#user

Reply:
Well, let us do that in the workflow as needed. It will make it clearer.